Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -16,10 +16,19 @@
             </intent-filter>
         </activity>
         <meta-data android:name="unityplayer.SkipPermissionsDialog" android:value="false" />
-        <meta-data android:name="pvr.app.type" android:value="vr" />
     </application>
-    <uses-permission android:name="android.permission.RECORD_AUDIO" tools:node="remove"/>
-    <uses-permission android:name="android.permission.READ_PHONE_STATE" tools:node="remove"/>
+
+    <meta-data android:name="com.oculus.vr.focusaware" android:value="true"/>
+    <meta-data android:name="com.oculus.supportedDevices" android:value="quest|quest2" />
+    <meta-data android:name="pvr.app.type" android:value="vr" />
```

Comment:
did you mean to move this outside of `<application>`?

Reply:
I actually meant to remove it entirely, my plugin can handle it now! Keep this open as a reminder